### DISCUSSION TITLE
希望深度提供在双显示器之间移动应用程序窗口的快捷键操作

### DISCUSSION THREAD

**Discussion by bigben446** [linuxdeepin/developer-center#4090] [Features Request & Ideas | 特性请求 & 头脑风暴]
这是深度论坛上的一个帖子，发帖者很可怜，从2018年就开始提建议，提了五次，都没有得到解决。关键是对于开发者来说，解决起来一点都不难，ubuntu/gnome都已经实现了。难的是开发者对**这种技术难度为零但是非常影响用户体验的问题**加以重视。

原贴地址在
https://bbs.deepin.org/zh/post/245411?offset=0&postId=1451317

希望深度提供在双显示器之间移动应用程序窗口的快捷键操作
平时工作使用的是双显示器，所以特别希望深度提供在双显示器之间移动应用程序窗口的快捷键操作。
在 windows10 和 ubuntu/gnome中，对双显示器支持很好，并且提供了快捷键操作。按下  Win+Shift+右箭头，可以把当前应用程序窗口移动到右侧的显示器，按下  Win+Shift+左箭头
可以把当前应用程序窗口移动到左侧的显示器。看起来是个不经意的事情，却给双显示器用户带来极大的方便，希望深度也能提供类似的功能。
这个功能主流的桌面环境都默认使用Super+Shift+光标箭头来移动，包括Windows、gnome、kde、budige、mate等等，都使用
[Win+Shift+←] 或者  [Win+Shift+→]，来回在左右两个显示器间移动应用程序窗口。

最早在2018年就提出这个建议：
https://bbs.deepin.org/zh/post/180522

后来在2019年又再次提出：
https://bbs.deepin.org/zh/post/166889

在2021年11月5日再次提出：
https://bbs.deepin.org/post/227694

在2022年2月23日再次提出
https://bbs.deepin.org/post/232122

官方答复说正在开发，后来看到开发了一个快捷键调用双屏设置的功能，而不是快捷键在双屏幕之间移动应用程序窗口。

双屏设置，会在使用笔记本连接投影仪或电视大屏幕时候使用，使用频度总是有限的。

而快捷键在双屏幕之间移动应用程序窗口，对于双显示器用户来说，确实每天每小时都经常进行的操作。

今天是2022年11月5日，再开新帖，希望尽快实现这个功能！

**Comment by BLumia** [comment]
> 在双显示器之间移动应用程序窗口的快捷键操作

我目前没双屏设备所以大概没法测试，但可能可以通过 systemsettings 配置 kwin 的相关快捷键？不太确定。cc @justforlxz 

> 这种技术难度为零但是非常影响用户体验的问题

如果你已经知道解决方案的话，欢迎直接发起 PR
